### PR TITLE
AJ-688: Remove isFeaturePreview for WDS now that Leo supports WDS

### PR DESCRIPTION
**Pull request by aaronkanzer**
Relates to https://broadworkbench.atlassian.net/browse/AJ-688

WDS for Azure workspaces is now supported by Leo, thus, we can now remove the `Feature Preview` requirement 🎉 

This PR removes references to that logic in the UI

### CODE REVIEW

**Review by calypsomatic — APPROVED** [review]

File: src/pages/workspaces/workspace/Data.js [hunk 1]
```diff
@@ -671,7 +671,7 @@ const WorkspaceData = _.flow(
 
   return div({ style: styles.tableContainer }, [
     !entityMetadata ? spinnerOverlay : h(Fragment, [
-      (isGoogleWorkspace || isFeaturePreviewEnabled('workspace-data-service')) && div({ style: { ...styles.sidebarContainer, width: sidebarWidth } }, [
+      isGoogleWorkspace && div({ style: { ...styles.sidebarContainer, width: sidebarWidth } }, [
```
Comment (jordansafer) [inline src/pages/workspaces/workspace/Data.js]: I think we want this for WDS workspaces too, it looks like we'll only it show it for google workspaces with the current logic-

File: src/pages/workspaces/workspace/WorkspaceContainer.js [hunk 2]
```diff
@@ -95,7 +94,7 @@ const WorkspaceTabs = ({
 
   const tabs = [
     { name: 'dashboard', link: 'workspace-dashboard' },
-    ...(isFeaturePreviewEnabled('workspace-data-service') || isGoogleWorkspace ? [{ name: 'data', link: 'workspace-data' }] : []),
```
Comment (jordansafer) [inline src/pages/workspaces/workspace/WorkspaceContainer.js:98]: Similar comment here. I think we do want to have the data tab for azure workspaces

File: src/pages/workspaces/workspace/Data.js (hunk 1 above)
Comment (aaronkanzer) [reply src/pages/workspaces/workspace/Data.js]: Updated!

File: src/pages/workspaces/workspace/WorkspaceContainer.js (hunk 2 above)
Comment (aaronkanzer) [reply src/pages/workspaces/workspace/WorkspaceContainer.js:98]: Updated!

**Review by jordansafer — APPROVED** [review]
Looks good! I know in one place there's a check for non-Prod: `    if (!getConfig().isProd && isAzureWorkspace) { ...` . I believe since right now TOA is still in private preview, there isn't much of a distinction here, so this shouldn't make much difference. In the future if we have parts of WDS we are trying to hide from prod post release, we should re-evaluate!

File: src/pages/workspaces/workspace/Data.js [hunk 3]
```diff
@@ -566,7 +566,7 @@ const WorkspaceData = _.flow(
   }
 
   const loadWdsSchemaAndUrl = async () => {
-    if (isFeaturePreviewEnabled('workspace-data-service') && !getConfig().isProd && isAzureWorkspace) {
+    if (!getConfig().isProd && isAzureWorkspace) {
```
Comment (davidangb) [inline src/pages/workspaces/workspace/Data.js:569]: echoing Jordan's comment here, we'll need to remove the `isProd` check at some point. I suggest we make a new ticket and do this in January when everybody is back to work.

**Review by davidangb — COMMENTED** [review]
if we're removing all checks for this feature flag, how about removing the feature flag itself?
https://github.com/DataBiosphere/terra-ui/blob/02959aec2697962cfe20aa33643c85ee53a14a88/src/libs/feature-previews-config.js#L16-L22

We'll also need to make minor updates to our "getting started" doc to remove references to this feature flag.

File: src/pages/workspaces/workspace/Data.js (hunk 3 above)
Comment (aaronkanzer) [reply src/pages/workspaces/workspace/Data.js:569]: Sounds good -- made a ticket! https://broadworkbench.atlassian.net/browse/AJ-757

**Comment by aaronkanzer** [comment]
> if we're removing all checks for this feature flag, how about removing the feature flag itself?
> 
> https://github.com/DataBiosphere/terra-ui/blob/02959aec2697962cfe20aa33643c85ee53a14a88/src/libs/feature-previews-config.js#L16-L22
> 
> We'll also need to make minor updates to our "getting started" doc to remove references to this feature flag.

Removed the feature flag!

I think I need `edit` access to this doc to update the directions: https://docs.google.com/document/d/1lWoHKJjISOpFzNWr6DeABYxUpp-uxMgtO3FPoirUc9w/comment#

**Review by davidangb — APPROVED** [review]
thank you! Beth can update the getting-started instructions - and we probably want to coordinate that edit with when this PR actually gets released to the BEE

**Comment by aaronkanzer** [comment]
> thank you! Beth can update the getting-started instructions - and we probably want to coordinate that edit with when this PR actually gets released to the BEE

I ended up updating the doc!

Cc @bethsheets

**Review by aherbst-broad — APPROVED** [review]

**Pull request merged**